addons/manifests/cni-webhook-manifests.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
```

Comment:
name should be `tanzu-addons-manager-webhook-service`

Reply:
Fixed